test/models/hold_test.rb

```diff
@@ -70,6 +70,33 @@ class HoldTest < ActiveSupport::TestCase
     assert(hold.invalid?)
   end
 
+  test 'active_or_expired scope shows expected records' do
```

Comment:
It might be easier to read these if they were separate tests:

`active_or_expired scope only shows active and expired records`
`active_or_expired scope returns expected active record`
`active_or_expired scope returns expected expired record`
`active_or_expired scope does not return active record`